runtime/rococo/src/lib.rs

```diff
@@ -245,6 +248,13 @@ construct_runtime! {
 		// Validator Manager pallet.
 		ValidatorManager: validator_manager::{Pallet, Call, Storage, Event<T>},
 
+		// Bridge messages support. The same story as with the bridge grandpa pallet above ^^^ - when we're
+		// running as Rococo we only use `BridgeWococoMessages`/`BridgeWococoMessagesDispatch`, and vice versa.
+		BridgeRococoMessages: pallet_bridge_messages::{Pallet, Call, Storage, Event<T>, Config<T>} = 43,
```

Comment:
It's a little unfortunate that we're being split by `ValidatorManager`, do you think anything (major) would break if we moved it after the message lane pallets 👀

Reply:
It requires a transaction spec version update. I personally don't care about this split

Reply:
I think it would nice to have all the bridge stuff side by side, but I won't push too hard on this

Reply:
Yeah, I agree Hernando here. @shawntabrizi do you mind if we change the pallet index of `ValidatorManager` in the next restart/runtime upgrade of Rococo? I'd move it above the bridge stuff.